src/mod/l2_debug_com.h

```diff
@@ -15,7 +15,7 @@
 #include "ado_modules.h"
 
 // RX definitions
-#define DEB_L2_MAX_CHARPERLINE		100
+#define DEB_L2_MAX_CHARPERLINE		10
```

Comment:
Was this changed intentional!?  I remember this is the input buffer for a single command line and there should be more room than 10 char, especially when we allow 10 args (with the next define!) ??

Reply:
This was not mine change. I believe it was from develop branch.

Reply:
ok never mind where it comes from. Change back to 100 pls.

Reply:
https://github.com/RobertK66/obc_1769_core/pull/34/commits/841628fe94e1916b2ffdb5d92fc97200093c1b6a

Fixed